src/main/java/net/minecraftforge/client/event/ScreenEvent.java

```diff
@@ -45,24 +43,24 @@
  * @author bspkrs
  */
 @OnlyIn(Dist.CLIENT)
-public class GuiScreenEvent extends Event
+public class ScreenEvent extends Event
 {
     private final Screen gui;
 
-    public GuiScreenEvent(Screen gui)
+    public ScreenEvent(Screen gui)
     {
         this.gui = gui;
     }
 
     /**
      * The GuiScreen object generating this event.
```

Comment:
Reference to GuiScreen

Reply:
See my previous comment: https://github.com/MinecraftForge/MinecraftForge/pull/7255#discussion_r471100490.